tests/test_text_onnx_embeddings.py

```diff
@@ -27,6 +27,8 @@
     ),
     "thenlper/gte-large": np.array([-0.01920587, 0.00113156, -0.00708992, -0.00632304, -0.04025577]),
     "mixedbread-ai/mxbai-embed-large-v1": np.array([0.02295546, 0.03196154, 0.016512, -0.04031524, -0.0219634]),
+    "intfloat/multilingual-e5-large-instruct": np.array([-0.00201539 , 0.04259416, -0.01194609, -0.04797889, -0.00045431]),
```

Comment:
This is wrong. The vector values for the `docs` mentioned in tests are different. Your [Colab Notebook](https://colab.research.google.com/drive/1E9s8UiBT4q1cr76Wlk7gP4bIkeCXJ-LZ#scrollTo=spJiCcNWs6-O) is largely correct — just use the same docs as tests.

Reply:
Thanks for mentioning this error, I have  updated the `canonical_vector_values` as per the docs.